modules/ROOT/partials/integrations/react-quick-start.adoc

```diff
@@ -236,6 +241,11 @@ import 'tinymce/plugins/wordcount';
 // importing plugin resources
 import 'tinymce/plugins/emoticons/js/emojis';
 
+// Import premium plugins from NPM (optional, requires tinymce-premium package)
+import 'tinymce-premium/plugins/advcode';
+import 'tinymce-premium/plugins/licensekeymanager';
```

Comment:
Should we note that the license key manager is required to use any premium plugins in places such as this, or leave that detail for other pages?